clowdapp.yaml

```diff
@@ -80,19 +80,99 @@ objects:
           failureThreshold: 3
           httpGet:
             path: /
-            port: 8000
+            port: 9000
             scheme: HTTP
-          initialDelaySeconds: 30
-          periodSeconds: 10
+          initialDelaySeconds: 10
```

Comment:
Would like to know why we changed `initialDelaySeconds` to 10 instead of 30?

Reply:
No strong reason, I just thought that delay of 30 second is to much and thought to match it with the other services on consoledot.